provisioner/powershell/provisioner_test.go

```diff
@@ -849,3 +852,12 @@ func testConfigWithSkipClean() map[string]interface{} {
 		"skip_clean": true,
 	}
 }
+
+func generatedData() map[string]interface{} {
+	generatedData := map[string]interface{}{
+		"PackerHTTPAddr": "",
```

Comment:
More realistic test would be "ERR_HTTP_ADDR_NOT_IMPLEMENTED_BY_BUILDER", right?

Reply:
ops, you're right!!